README.md

```diff
@@ -0,0 +1,28 @@
+# NLQ Notebooks
```

Comment:
Let's also link to the LICENSE file explicitly in a License section. `This project is licensed under ... `

Reply:
Done, tnx